### ISSUE TITLE
Remove reserved keyword for PHP-7.1.0

### ISSUE THREAD

**Issue by vonglasow** [atoum/atoum#598]
Apparently `void` should be a reserved keyword in PHP-7.1.0

example: https://continuousphp.com/git-hub/vonglasow/praspel/build/c6c9e55e-3609-4977-bfd4-c78343451277/output

=> see output for the version PHP-7.1.0
To check with the branch master of PHP. (No tag found)

**Comment by remicollet** [comment]
Indeed:

```
+ php71 runner.php --directories .
Fatal error: Cannot use 'void' as class name as it is reserved in /dev/shm/BUILD/atoum-a3bceadf39cccd4cc321407eeb61187bdee438cd/classes/report/fields/runner/tests/void.php on line 11

```

**Comment by Hywan** [comment]
Official source: https://github.com/php/php-src/blob/php-7.1.0alpha2/UPGRADING.

**Comment by vonglasow** [comment]
To complete this issue:
- [ ] replace void by blank
- [ ] complete changelog
- [ ] add alias in autoloading

**Comment by Hywan** [comment]
Please, update the `CHANGELOG` and we are good :-).

**Comment by Hywan** [comment]
It's like we are good now. Please @vonglasow, can you write the `CHANGELOG` entry?

**Comment by vonglasow** [comment]
CHANGELOG entry added!

**Comment by mvrhov** [comment]
When will this get merged? I have an php 7.1 only application and I'm unable to run tests.

**Comment by jubianchi** [comment]
#615 was merge. 3.0.0 release will be coming soon!
